libraries/app/application.cpp

```diff
@@ -989,9 +992,9 @@ void application::initialize(const fc::path& data_dir, const boost::program_opti
                return;
          }
 
-         std::cerr << "Updating genesis state in file " << genesis_out.generic_string() << "\n";
+         ilog("Updating genesis state in file ${genesis_out} \n", ("genesis_out", genesis_out.generic_string()));
```

Comment:
I don't think here and below should be changed to `ilog`. @pmconrad what's your opinion?